Sources/ComposableArchitecture/UIKit/ActionSheetStateUIKit.swift

```diff
@@ -0,0 +1,27 @@
+#if canImport(UIKit) && !os(watchOS)
+import UIKit
+
+@available(iOS 13, *)
+@available(macCatalyst 13, *)
+@available(macOS, unavailable)
+@available(tvOS 13, *)
+@available(watchOS, unavailable)
+extension ActionSheetState {
+  /// UIKit helper generating UIAlertController from corresponding ActionSheetState
+  /// - Parameter send: A function that wraps a alert action in the view store's action type.
+  /// - Returns: UIAlertController ready to presented by UIViewController.
+  public func toUIAlertController(send: @escaping (Action) -> Void) -> UIAlertController {
```

Comment:
The main change we think is API. This looks a lot like the `toSwiftUI` stuff we define [here](https://github.com/pointfreeco/swift-composable-architecture/blob/a4f544e892f04822def9f5934ff4d7ecfdf1329e/Sources/ComposableArchitecture/SwiftUI/Alert.swift#L244), but that naming is an internal detail and not reflected in the public-facing API.

What do you think of adding this functionality as a convenience initializer on `UIAlertController` instead? Something like:

```swift
UIAlertController(state: AlertState, send: (Action) -> Void)
```